test/integration/targets/collections/runme.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -eux
+
+export ANSIBLE_INSTALLED_CONTENT_ROOTS=$PWD/collection_root_user,$PWD/collection_root_sys
+export ANSIBLE_GATHERING=explicit
+export ANSIBLE_GATHER_SUBSET=minimal
+
+ansible-playbook -i ../../inventory -i ./a.statichost.yml -v play.yml
```

Comment:
I think it would be useful to add another invocation of ansible-playbook to the tests, but with config/inventory setup so that nothing causes early plugin loading (ie, stuff like specifying a callback in the config, or the inventory plugin used in    test/integration/targets/collections/a.statichost.yml). 

The goal would be to catch any issues that occur if a collection loader hasn't been created early in the process. Based on some of the unexpected and hard to debug behavior seen in earlier versions, some tests for the 'no early plugin or collection loader' cases would be useful.